Enterprise-CMCS/macpro-mdct-mfp
# Setup for Reports and Admin Banner Stores

### Description

Creating the stores, initial states, and actions for reports and admin banners, as well as a global store.

### Related ticket(s)



---
### How to test

This one is a bit difficult to test, mostly it'll be for the `userStore` at this point, but once plug in all of the API calls and whatnot, the result should be a bit clearer.

If you log in as any user and use the Redux DevTools extension, you should see something like this:

![Screenshot 2023-08-30 at 5 24 56 PM](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/99458559/db7e065c-dde0-4358-8db7-2a20469ba9df)

The global state of the current user you're logged in as!

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
